guides/proxy-protocol.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: doc
+sublayout: guide
+title: Running Quay.io behind a load balancer
+---
+
+Running an enterprise instance of Quay.io behind a load balancer is often desired for large installations.
+However, simply putting a load balancer in front of the Quay.io container has an unintended consequence: all logged IP addresses will be the IP of the load balancer.
+This may not break any major functionality, but it can be avoided by using [Proxy Protocol](http://www.haproxy.org/download/1.5/doc/proxy-protocol.txt) which is exposed on **port 8443**.
```

Comment:
This is pretty ambiguous. It may break major functionality? It will break minor functionality? What am I risking by not adopting this?